mk/checkconf.mk

```diff
@@ -26,6 +26,18 @@ define check-conf-h
 	$(call mv-if-changed,$@.tmp,$@)
 endef
 
+define check-conf-cmake
+	$(q)set -e;						\
+	$(cmd-echo-silent) '  CHK     $@';			\
+	cnf='$(strip $(foreach var,				\
+		$(call cfg-vars-by-prefix,$1),			\
+		$(call cfg-cmake-set,$(var))))';		\
+	guard="_`echo $@ | tr -- -/. ___`_";			\
+	mkdir -p $(dir $@);					\
+	echo -n "$${cnf}" | sed 's/_nl_ */\n/g' >>$@.tmp;	\
```

Comment:
Why not keep the "auto-generated..." text from `check-conf-mk`? Since this file is used externally, it's good to know which version has created it.